samples/04_gis_analysts_data_scientists/determining_site_suitability_for_oil_palm_plantation.ipynb

```diff
@@ -0,0 +1,639 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    upload = arcgis.gis.ContentManager(gis)
</pre><p>Can we simplify the first two lines into this:</p><p><code>gis.content.add(item_properties={....})</code>?</p><p> </p><p>For notebooks that are samples, we typically provide a reproducible workflow. Hence, can you zip up and upload the rasters, RFT as an item on Geosaurus AGO org and refer to it in the notebook? Perhaps you can add a cell before this which downloads this item, unpacks (using zipfile lib) to a relative path which is fed to each of the copy_raster and this add_item call?</p><p> </p><p>You can publish the item using <code>api_data_owner</code>  account, delete protect it and share it to <code>everyone</code>  and refer to it using its item id</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>